nifi-nar-bundles/nifi-standard-services/nifi-record-serialization-services-bundle/nifi-record-serialization-services/src/test/resources/xml/person_record.xml

```diff
@@ -0,0 +1,5 @@
+<record>
```

Comment:
Please add this file to the RAT exclusions

Reply:
We should avoid RAT exclusions if possible. Any file that can have a proper header that includes the Apache license info should. RAT exclusions should only be used for binary files, etc. where a proper license can't be included.